assets/src/styles/campaigns/themes/_theme_arctic.scss

```diff
@@ -320,4 +320,32 @@ body.theme-arctic {
   .video-block {
     @include campaign_video($arctic-light-blue);
   }
+
+  .page-template table.spreadsheet-table {
+    color: #fff;
+
+    tr {
+      background-color: #8D9C9E;
```

Comment:
These still need to be moved to the JSON files, but I want to review the POC early.